src/modules/SuSEFirewallCMDLine.rb

```diff
@@ -1599,6 +1634,40 @@ def Run
       nil
     end
 
+  private
+    # Returns true if FirewallD is the running backend
+    def firewalld?
+      return true if SuSEFirewall.is_a?(Yast::SuSEFirewalldClass)
+
+      false
+    end
+
+    def ConfigureFirewalld
+      return if !firewalld?
```

Comment:
in ruby is convention to use unless for single statement trailing condition, it looks like `return unless firewalld?`